app/renderer/components/preferences/payment/enabledContent.js

```diff
@@ -310,6 +337,75 @@ const styles = StyleSheet.create({
     ':hover': {
       textDecoration: 'none !important'
     }
+  },
+
+  enabledContent: {
+    position: 'relative',
+    zIndex: 2
+  },
+
+  loader: {
+    background: '#fafafa',
+    zIndex: 3,
+    position: 'absolute',
+    left: 0,
+    top: 0,
+    width: '100%',
+    height: 0,
+    margin: 0,
+    opacity: 0,
+    transition: 'opacity .4s ease-out, height .1s .4s ease'
+  },
+
+  loader_show: {
+    opacity: 1,
+    height: '100%',
+    transition: 'opacity .4s ease-out'
+  },
+
+  loader__text: {
+    textAlign: 'center',
+    padding: '50px 0 20px',
+    display: 'block',
+    color: '#444'
+  },
+
+  leader__wrap: {
+    width: '45px',
+    left: 0,
+    right: 0,
+    margin: '50px auto 0'
+  },
+
+  loader__line: {
+    display: 'inline-block',
+    width: '15px',
+    height: '15px',
+    borderRadius: '15px'
+  },
+
+  loader__line_1: {
+    backgroundColor: '#FF5000',
+    animationName: [loaderAnimation],
+    animationDuration: '.6s',
+    animationDelay: '.1s',
+    animationIterationCount: 'infinite'
```

Comment:
please don't forget to dedupe the animation properties later

Reply:
done

Reply:
````
loader__line: {
     display: 'inline-block',
     width: '15px',
     height: '15px',
     borderRadius: '15px',
     animationName: [loaderAnimation],
     animationDuration: '.6s',
     animationIterationCount: 'infinite'
  },

  loader__line_1: {
     backgroundColor: '#FF5000',
     animationDelay: '.1s'
}
````

and so on

Reply:
done